Vintage_effect.py

```diff
@@ -0,0 +1,29 @@
+# Import the dependencies
+import cv2
+import argparse
+import numpy as np
+
+# construct the argument parse and parse the arguments
+ap = argparse.ArgumentParser()
+ap.add_argument("-i", "--image", required=True, help="path to input image")
+args = vars(ap.parse_args())
+
+# reading the image
+img = cv2.imread((args["image"]))
+
+rows, cols = img.shape[:2]
+

```

Comment:
- Put the entire filter inside a function.
- Split the image into multiple channels
- Call the function for each of the three channels
- Merge them